certif/app/controllers/terms-of-service.js

```diff
@@ -4,13 +4,18 @@ import { inject as service } from '@ember/service';
 
 export default class TermsOfServiceController extends Controller {
   @service currentUser;
+  @service notifications;
 
   @action
   async submit() {
-    await this.currentUser.certificationPointOfContact.save({
-      adapterOptions: { acceptPixCertifTermsOfService: true },
-    });
-    this.currentUser.certificationPointOfContact.pixCertifTermsOfServiceAccepted = true;
-    return this.transitionToRoute('authenticated.sessions.list');
+    try {
+      await this.currentUser.certificationPointOfContact.save({
+        adapterOptions: { acceptPixCertifTermsOfService: true },
+      });
+      this.currentUser.certificationPointOfContact.pixCertifTermsOfServiceAccepted = true;
```

Comment:
Remarque, j'ai pas eu besoin de faire ça pour la connexion.
l'apple à http://localhost:4203/api/users/104439/pix-certif-terms-of-service-acceptance met les cgu à true
je suis pas allée voir la page cgu mais elle fonctionne déjà pour les utilisateurs pix certif (si un utilisateur se loggue et que ses cgu certif sont à false, il passe par cette page).
je pense que tu peux supprimer ce bout de code 
sauf si tu as vu un cas ?

Reply:
Cet appel retourne une 204. Le modèle `CertificationPointOfContact` ne sera pas modifié au niveau du front. Je pense que ce code est nécessaire.

Pour ce fichier, nous avons juste englobé le code dans un `try-catch` pour pouvoir afficher une notification en cas d'erreur.

Reply:
Ah mais oui, j'avais pas bien vu, je retire ce que j'ait dit